css/css-backgrounds/parsing/background-valid.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="utf-8">
+<title>CSS Backgrounds and Borders Module Level 3: parsing background with valid values</title>
+<link rel="author" title="Eric Willigers" href="mailto:ericwilligers@chromium.org">
+<link rel="help" href="https://drafts.csswg.org/css-backgrounds/#background">
+<meta name="assert" content="background supports the full grammar '<bg-layer># , <final-bg-layer>'.">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="resources/parsing-testcommon.js"></script>
+</head>
+<body>
+<script>
+// For <final-bg-layer>, implementations serialize <‘background-color’> after <bg-image>, contrary to spec.
+// Safari removes quotes.
```

Comment:
Hmm, does safari remove quotes unconditionally? How does it handle escaping? That's maybe worth a bug.

Reply:
https://github.com/w3c/csswg-drafts/issues/2547
https://bugs.webkit.org/show_bug.cgi?id=28869